controllers/users.js

```diff
@@ -23,6 +23,27 @@ class Users {
     const { email, username, password: hash } = req.body;
 
     try {
+      // check if there is user who is authenticated and if he is a super-admin and activate auser
+      if (req.user !== undefined && req.user.role === 'super-admin') {
+        const userCreate = await User.create({
+          email: email.trim(),
+          username: username.trim(),
+          hash,
+          following: JSON.stringify({ ids: [] }),
+          followers: JSON.stringify({ ids: [] }),
+          activated: 1
+        });
+        return res.status(201).json({
+          status: 201,
```

Comment:
it would better if you send an email to the user, give him option to change password.